File: ios/ShareExtensionTipManager.m
//
//  ShareExtensionTip.m
//  Mello
//
//  Created by Andrei on 2019/2/10.
//  Copyright © 2019 Facebook. All rights reserved.
//

//#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(ShareExtensionTipManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(onDismiss, RCTDirectEventBlock)
@end

File: ios/ShareExtensionTipManager.swift
//
//  ShareExtensionTip.swift
//  Mello
//
//  Created by Andrei on 2019/2/10.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation


@objc(ShareExtensionTipManager)
class ShareExtensionTipManager: RCTViewManager {
  override func view() -> UIView! {
    return ShareExtensionTip()
  }
  // this is required since RN 0.49+
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  // @objc func updateFromManager(_ node: NSNumber, count: NSNumber) {
  //   DispatchQueue.main.async {
  //     let component = self.bridge.uiManager.view(
  //       forReactTag: node
  //       ) as! ShareExtensionTip
  //     component.update(value: count)
  //   }
  // }
 
}
